### PR TITLE
Issue 24 - Method to add multiple EventsOfInterest to the model

### PR DESCRIPTION
**Pull request by amol-anand**
## Description
Need this for someone who wants to register multiple combinations of Provider ID and Event Code to one Journal/Webhook.

## Related Issue

https://github.com/adobe/aio-lib-java/issues/24

## Motivation and Context

Need this for someone who wants to register multiple combinations of Provider ID and Event Code to one Journal/Webhook.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [X] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [X] My code follows the [code style](CODE_STYLE.md) of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.

### CODE REVIEW

**Comment by amol-anand** [comment]
@francoisledroff  please review and merge. need a way to add multiple combos of ProviderId and EventCode while registering a new journal to listen to events.

File: events_mgmt/src/main/java/com/adobe/event/management/model/RegistrationInputModel.java [hunk 1]
```diff
@@ -157,6 +157,12 @@ public Builder addEventsOfInterests(
       return this;
     }
 
+    public Builder addMultipleEventsOfInterests(
```
Comment (francoisledroff) [inline events_mgmt/src/main/java/com/adobe/event/management/model/RegistrationInputModel.java:160]: wdyt just reusing the same method name `addEventsOfInterests` here ?

Comment (amol-anand) [reply events_mgmt/src/main/java/com/adobe/event/management/model/RegistrationInputModel.java:160]: makes sense considering the method name is plural and intended for that purpose. ill update the PR

**Comment by amol-anand** [comment]
@francoisledroff updated. please review

**Pull request merged**